src/pages/createEdit/Activity.tsx

```diff
@@ -14,7 +26,7 @@ import taskJson from "../../resources/default_layout/task.json";
 import eventJson from "../../resources/default_layout/event.json";
 import { FieldMappingInput } from "../../components/FieldMappingInput/FieldMappingInput";
 import { getMetadataBasedSchema } from "../../schemas/default";
-import { mapErrorMessage } from "../../utils";
+import { buttonLabels, capitalizeFirstLetter, mapErrorMessage } from "../../utils";
```

Comment:
`capitalizeFirstLetter` - `lodash` installed here, so you can use the [capitalize](https://lodash.com/docs/4.17.15#capitalize) function